src/frontend/src/pages/Login/Login.tsx

```diff
@@ -1,9 +1,27 @@
 import { Auth } from '@supabase/auth-ui-react';
 import { ThemeSupa } from '@supabase/auth-ui-shared';
+import useAuth from 'hooks/AuthHooks';
 import { supabase } from 'main';
+import AppRoutes from 'navigation/AppRoutes';
+import { Navigate } from 'react-router-dom';
 
 function Login() {
-    return <Auth supabaseClient={supabase} appearance={{ theme: ThemeSupa }} providers={[]}></Auth>;
+    const { session, isLoading } = useAuth();
+    // I want auto redirect but it's not working
+    if (!session) {
+        return (
+            <div className='loginClass'>
```

Comment:
Perhaps another name would be more descriptive than `loginClass`,

```suggestion
            <div className='login-form'>
```